tao_compiler/mlir/disc/tests/tensorflow_ops/sum.cc

```diff
@@ -33,6 +33,30 @@ TEST(TFSumOpTest, ColReduceFullyDynamicShape2DF32) {
       /*output_descriptors*/ {"f32_X"}));
 }
 
+// dynamic shape 2D column reduction test case
+TEST(TFSumOpTest, ColReduceFullyDynamicShape2DF16) {
+  EXPECT_TRUE(feature_test_main(
+      /*mlir_file_path*/ c_ft_path + "sum_col_d_2d_f16.mlir",
+      /*backend_types*/
+      kSupportedBackendList,
+      /*num_inputs*/ 1,
+      /*num_outputs*/ 1,
+      /*input_descriptors*/ {"110x100xf16_X"},
```

Comment:
cpu backend support f16？

Reply:
I have no idea whether the CPU backend supports fp16, but the UT passed.